termwiz/src/input.rs

```diff
@@ -365,6 +365,7 @@ impl KeyCode {
             }
 
             Tab => {
+                let mods = mods.remove_positional_mods();
```

Comment:
Hmm! Good find! However, I think this should probably be moved up to line 265 so that it applies to everything in this function, and not just Tab!